### PR TITLE
More logging for bonds compute

### PR DESCRIPTION
**Pull request by nzpr**
More logging for bonds compute

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by 9rb — APPROVED** [review]
lgtm

File: casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala [hunk 1]
```diff
@@ -731,7 +731,7 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
       .ensureOr(
         validatorsPar =>
           new IllegalArgumentException(
-            s"Incorrect number of results from query of current active balidator: ${validatorsPar.size}"
+            s"Incorrect number of results from query of current active validator in state ${startHash}: ${validatorsPar.size}"
```
Comment (tgrospic) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: Do we need `PrettyPrinter.buildString` for `startHash`? (the same in computeBonds)

Comment (nzpr) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala]: Yes, you are right!

**Review by tgrospic — APPROVED** [review]
LGTM

**Comment by nzpr** [comment]
bors merge

**Pull request merged**